README.md

```diff
@@ -10,14 +10,19 @@ Service APIs implemented:
 - Amazon SimpleDB
 - Amazon Mechanical Turk
 - Amazon CloudWatch (MON)
-- Amazon DynamoDB (ddb2) 
+- Amazon DynamoDB (ddb2)
 
-All API functions have been implemented.  Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.  Please send bug reports and patches.
+All API functions have been implemented.  Not all functions have been
```

Comment:
it looks like this was done by some tool like a plugin to iDEA and clearly it's broken as this is not what .md files need.